influxdb3_write/src/catalog.rs

```diff
@@ -57,15 +82,36 @@ impl Catalog {
             return Err(Error::CatalogUpdatedElsewhere);
         }
 
-        info!("inserted/updated database in catalog: {}", db.name);
+        // Check we havenot gone over the table limit with this updated DB
```

Comment:
👋 Longtime lurker, first time commenter with a quick typo comment 😄 

```suggestion
        // Check we have not gone over the table limit with this updated DB
```

Reply:
Thanks for the find! 😁